mass_mailing_list_dynamic/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,6 @@
+Without this addon you have to choose between providing a dynamic domain and
+letting your mass mailings reach all partners that match it, or being able to
+unsubscribe to certain mailing lists while still being subscribed to others.
```

Comment:
`...or being able to
unsubscribe to certain mailing lists while still being subscribed to others. ...`

This behavior is already incorporated in the mass_mailing module of odoo 12